client/src/components/CustomTables.js

```diff
@@ -173,7 +177,40 @@ const upcomingOrOngoingTableStyles = makeStyles({
 export function UpcomingOrOngoingTable(props) {
   const classes = upcomingOrOngoingTableStyles();
   const [ongoingInterviewList, setOngoingInterviewList] = useState([]);
-
+  const { user } = useContext(UserContext);
+  const [inCodeList, setinCodeList] = useState([]);
+  const history = useHistory();
+  function cancelInterview(id) {
+    fetch("interviews/remove", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ id: id }),
+    })
+      .then(() => {
+        let arr = ongoingInterviewList;
+        let index = arr.indexOf(id);
+        if (index !== -1) {
+          arr.splice(index, 1);
+          setOngoingInterviewList([...arr]);
```

Comment:
I'd rather you change line 190 to: 
const arr = [... ongoingInterviewList]; 

A few points:
- It's bad practice to change ongoingInterviewList directly, and you are doing that since arrays/objects in javascript are like pointers 
- If you make arr a copy, then you don't need to make it a copy here
- You can use const, since even though you are modifying arr, since you are not modifying what arr is pointing to